.github/workflows/static-analysis.yml

```
@@ -4,4 +4,4 @@ on: push
 
 jobs:
   call-secrets-analysis-workflow:
-    uses: ASFHyP3/actions/.github/workflows/reusable-secrets-analysis.yml@v0.9.0
+    uses: ASFHyP3/actions/.github/workflows/reusable-secrets-analysis.yml@jhkennedy-patch-1
```

This change should be reverted before merging but allows us to test the changes here
```suggestion
    uses: ASFHyP3/actions/.github/workflows/reusable-secrets-analysis.yml@v0.9.0
```